ci/behat-tags.php

```diff
@@ -37,6 +37,7 @@ function version_tags( $prefix, $current, $operator = '<' ) {
 if ( ! in_array( getenv( 'WP_VERSION' ), array( 'latest', 'nightly', 'trunk' ), true ) ) {
 	$wp_version_reqs = version_tags( 'require-wp', getenv( 'WP_VERSION' ), '<' );
 }
+$wp_version_reqs = array_merge( $wp_version_reqs, version_tags( 'less-than-wp', getenv( 'WP_VERSION' ), '>' ) );
```

Comment:
I think this should be inside of the above conditional's block, to avoid comparisons against `latest`, `nightly` and `trunk`.

Reply:
Good point. Need also to always set `@less-than-wp` tags in those cases I think... hack incoming...